solutions/swb-reference/CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ Research Service Workbench (RSW) on AWS is a backend API designed to empower AWS
 
 For additional details please refer to the implementation guide [here](https://docs.aws.amazon.com/solutions/latest/research-service-workbench-on-aws/overview.html)
 
-`rsw-reference` is the entry point to Research Service Workbench. Customers can configure and deploy RSW with the provided CDK constructs and modules. 
+`swb-reference` is the entry point to Research Service Workbench. Customers can configure and deploy RSW with the provided CDK constructs and modules. 
```

Comment:
We're not renaming the folder structure to `rsw`?

Reply:
We didn't have enough time to do this refactor so we're leaving these as swb-reference and swb-app.